android/src/main/java/com/tailscale/ipn/ui/view/PeerDetails.kt

```diff
@@ -28,55 +28,73 @@ import androidx.compose.ui.platform.LocalClipboardManager
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.AnnotatedString
 import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
 import com.tailscale.ipn.R
 import com.tailscale.ipn.ui.util.settingsRowModifier
 import com.tailscale.ipn.ui.viewModel.PeerDetailsViewModel
+import com.tailscale.ipn.ui.viewModel.PeerDetailsViewModelFactory
 
 
 @Composable
-fun PeerDetails(viewModel: PeerDetailsViewModel) {
+fun PeerDetails(
```

Comment:
This isn't working.  On initial load, the peer is null and the netmap is null.   The netmap value on the model needs to be observed.

Reply:
This seems to be working now.